api/apps/geoprocessing/src/modules/planning-area/planning-area.geo.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  Check,
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { BBox, MultiPolygon } from 'geojson';
+import { defaultSrid } from '@marxan/utils/geo';
+
+@Entity('planning_area')
```

Comment:
Let's stick to plural table names (here and in the migration, if it is written manually, which I assume it is since we make use of generated columns).

```suggestion
@Entity('planning_areas')
```